paddlers/tasks/utils/slider_predict.py

```diff
@@ -298,10 +329,11 @@ def slider_predict(predict_func,
         invalid_value (int): Value that marks invalid pixels in output image. 
             Defaults to 255.
         merge_strategy (str): Strategy to merge overlapping blocks. Choices are 
-            {'keep_first', 'keep_last', 'accum'}. 'keep_first' and 'keep_last' 
+            {'keep_first', 'keep_last', 'accum', 'swell'}. 'keep_first' and 'keep_last' 
             means keeping the values of the first and the last block in 
             traversal order, respectively. 'accum' means determining the class 
             of an overlapping pixel according to accumulated probabilities.
+            'swell' means keep center block only.
```

Comment:
`keep center block only` -> `keeping only the center part of each block prediction`

Reply:
done